src/resolvers/types/deployment/index.js

```diff
@@ -87,19 +87,16 @@ export function type() {
 }
 
 /*
- * Return information about the deployment images.
- * @param {Object} parent The result of the parent resolver.
- * @return {Object} The deployment info.
+ * Return list tags from docker registry api.
+ * @return {Array} The deployment type.
  */
-export async function deployInfo(parent) {
-  // Build the repo name.
-  const repo = `${parent.releaseName}/${DEPLOYMENT_AIRFLOW}`;
-
+export async function registryTags(parent) {
   // Create a JWT for the registry request.
   const dockerJWT = await createDockerJWT("houston", [
     {
       type: "repository",
-      name: repo,
+      // Build the repo name.
+      name: `${parent.releaseName}/${DEPLOYMENT_AIRFLOW}`,
       actions: ["push", "pull"]
```

Comment:
Not that you changed this line, but while we are here: we don't need push permission to get the tags, so lets fix that:

```suggestion
      actions: ["pull"]
```